Resources/Private/Fusion/Forms/Validators/EmailAddress.ts2

```diff
@@ -1,6 +1,9 @@
 prototype(PackageFactory.AtomicFusion.Forms:Validators.EmailAddress) < prototype(PackageFactory.AtomicFusion.Forms:Validator) {
-    implementationClassName = 'Neos\\Flow\\Validation\\Validator\\EmailAddressValidator'
+    implementationClassName = 'PackageFactory\\AtomicFusion\\Forms\\Domain\\Model\\Validator\\FlowValidatorAdapter'
     options {
-        # @see http://neos.readthedocs.io/en/stable/References/Validators/Flow.html#emailaddressvalidator
+        validator = 'Neos\\Flow\\Validation\\Validator\\EmailAddressValidator'
+        options {
```

Comment:
Das war ein Überbleibsel ... es wird in der tat alles außer validator zum flowValidator übertragen